dkg/disk.go

```diff
@@ -53,6 +61,27 @@ func loadDefinition(conf Config) (cluster.Definition, error) {
 	return res, nil
 }
 
+// fetchDefinition fetches cluster definition file from a remote URI.
+func fetchDefinition(url string) (cluster.Definition, error) {
+	resp, err := http.Get(url) //nolint:gosec,noctx
```

Comment:
discuss: should i pass a `http.Client` as a function parameter?

Reply:
just pass in context and use a 5s timeout

Reply:
```
func fetchDefinition(ctx context.Context, url string) (cluster.Definition, error) {
    ctx, cancel := context.WithCancel(ctx, time.Second*5)
    defer cancel()
    
    req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
    
    resp, err = new(http.Client).Do(req)
```